### PR TITLE
update: supported version list

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

File: docs/topics/gradle.md [hunk 1]
```diff
@@ -1,41 +1,48 @@
...
 }
 ```
 
 </tab>
 </tabs>
 
-The placeholder `<...>` should be replaced with the name of one of the plugins that will be discussed in subsequent sections.
+When configuring your project, check the Kotlin Gradle plugin compatibility with available Gradle versions:
+
+|                       | Minimum supported version | Maximum fully supported version |
```
Comment (Tapchicoma) [inline docs/topics/gradle.md:39]: Probably we need to clarify for users what do we meant by "Maximum fully supported version".

File: docs/topics/gradle.md [hunk 2]
```diff
@@ -1,41 +1,48 @@
...
-<microformat>
-    <p>Minimum supported Gradle version: <strong>%minGradleVersion%</strong></p>
-    <p>Minimum supported Android Gradle plugin version: <strong>%minAndroidGradleVersion%</strong></p>
-</microformat>
+Gradle is a build system that helps to automate and manage your building process. It downloads specified dependencies,
+packages your code, and prepares it for compilation.
 
-In order to build a Kotlin project with Gradle, you should [apply the Kotlin Gradle plugin to your project](#plugin-and-versions)
-and [configure the dependencies](#configuring-dependencies).
+To build a Kotlin project with Gradle, you'll need to add a Gradle extension, [the Kotlin Gradle plugin](#apply-the-plugin),
```
Comment (Tapchicoma) [inline docs/topics/gradle.md:6]: replace `extension` with `plugin`.

`extension` in terms of Gradle - is an optional part of the plugin

File: docs/topics/gradle.md (hunk 1 above)
Comment (danil-pavlov) [reply docs/topics/gradle.md:39]: Expanded this section, please take a look

File: docs/topics/gradle.md [hunk 3]
```diff
@@ -1,41 +1,49 @@
...
+When configuring your project, check the Kotlin Gradle plugin compatibility with available Gradle versions:
+
+|                       | Minimum supported version | Maximum fully supported version |
+|-----------------------|---------------------------|---------------------------------|
+| Gradle                | %minGradleVersion%        | %maxGradleVersion%              |   
+| Android Gradle plugin | %minAndroidGradleVersion% | %maxAndroidGradleVersion%       |
+
+For example, the Kotlin Gradle plugin and the `kotlin-multiplatform` plugin %kotlinVersion% require Gradle
+%minGradleVersion% for your project to compile. In turn, the maximum fully version that doesn't have deprecated Gradle
+methods and properties and supports all the current Gradle features is %maxGradleVersion%.
```
Comment (Tapchicoma) [inline docs/topics/gradle.md:46]: Imho sentence is a little hard to understand due to it's structure. But I am not a native English speaker - so if it passes language review - I am fine with it.

**Review by Tapchicoma — APPROVED** [review]

**Pull request merged**